var/spack/repos/builtin/packages/py-onnx/package.py

```diff
@@ -33,6 +35,7 @@ class PyOnnx(PythonPackage):
 
     # CMakeLists.txt
     depends_on("cmake@3.1:", type="build")
+    depends_on("py-pybind11@2.2:", type=("build", "link"))
 
     # requirements.txt
     depends_on("py-setuptools@61:", type="build")
```

Comment:
Does the setuptools version still need to be bumped?

Reply:
Right, good catch. That got accidentally reverted by the rebase. Upstream [requires 64 too now](https://github.com/onnx/onnx/blob/main/pyproject.toml#L4).
```suggestion
    depends_on("py-setuptools@64:", type="build")
```